app/models/privacy_notification.rb

```diff
@@ -0,0 +1 @@
+class PrivacyNotification < ActiveRecord::Base; end
```

Comment:
Currently this model doesn't need to do anything clever. But if we/the petitions team want the email content to include a list of the petitions the signatories have signed, it may need to include something like [this](https://github.com/alphagov/e-petitions/blob/master/app/models/dissolution_notification.rb#L11).